backend-schedule/src/main/java/eu/socialedge/hermes/backend/schedule/domain/Schedule.java

```diff
@@ -78,6 +78,6 @@ public void removeTrip(Trip trip) {
     }
 
     public Collection<Trip> trips() {
```

Comment:
Make it List<Trip> to be consistent with other classes eg https://github.com/socialedge/hermes/blob/develop-v2/backend-transit/src/main/java/eu/socialedge/hermes/backend/transit/domain/Route.java#L104